src/workspace/mod.rs

```diff
@@ -1893,6 +1922,31 @@ impl WorkspaceDirectory {
   }
 }
 
+/// Reads compilerOptions from tsconfig.json file next to the package.json
+/// See https://github.com/denoland/deno/issues/28455#issuecomment-2734956368
+#[allow(clippy::disallowed_methods)]
+fn compiler_options_from_ts_config_next_to_pkg_json<TSys: FsRead>(
+  sys: &TSys,
+  pkg_json: &PackageJson,
+) -> Option<ParsedTsConfigOptions> {
+  let mut ts_config_path = pkg_json.path.clone();
+  ts_config_path.set_file_name("tsconfig.json");
```

Comment:
I think a clone can be avoided here by instead doing:

```suggestion
  let ts_config_path = pkg_json.path.with_file_name("tsconfig.json");
```

Reply:
Thanks!